extensions/ql-vscode/src/variant-analysis/variant-analysis-manager.ts

```diff
@@ -214,7 +215,51 @@ export class VariantAnalysisManager
   }
 
   private async runVariantAnalysisFromPublishedPack(): Promise<void> {
-    throw new Error("Command not yet implemented");
+    const language = await askForLanguage(this.cliServer);
+
+    const packName = `codeql/${language}-queries`;
+    const packDownloadResult = await this.cliServer.packDownload([packName]);
+    const downloadedPack = packDownloadResult.packs[0];
+
+    const packDir = `${packDownloadResult.packDir}/${downloadedPack.name}/${downloadedPack.version}`;
```

Comment:
Should we use `join` here to handle different path separators?

Reply:
Yes, definitely. Apologies I was a bit sloppy here and forgot about `join`.